example/bi-lstm-sort/sort_io.py

```diff
@@ -62,7 +63,7 @@ def default_gen_buckets(sentences, batch_size, the_vocab):
 
     tl = 0
     buckets = []
-    for l, n in len_dict.iteritems(): # TODO: There are better heuristic ways to do this    
+    for l, n in len_dict: # TODO: There are better heuristic ways to do this
```

Comment:
my bad.  should be fixed by using viewitems() from future.utils.   I will try to only use future and __future__ to maintain compatibility.